[2025-01-23 01:19:45] students_app - INFO - Логгер инициализирован. Проверка кодировки: русский текст
[2025-01-23 01:19:45] students_app - INFO - Запуск приложения
[2025-01-23 01:19:45] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:19:45] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:19:45] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:19:50] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:19:50] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:19:50] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:19:55] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:19:55] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:19:55] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:20:00] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:00] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:00] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:20:05] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:05] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:05] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:20:10] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:10] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:10] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:20:15] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:15] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:15] students_app.windows.login - DEBUG - Соединение установлено успешно
[2025-01-23 01:20:20] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:20] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:22] students_app.windows.login - ERROR - Ошибка подключения к 192.168.0.164:9999: timed out
[2025-01-23 01:20:22] students_app.windows.login - ERROR - Подробная информация об ошибке:
Traceback (most recent call last):
  File "c:\Users\MrDes\OneDrive\Документы\GitHub\Pselu--Pself-\students_app\windows\login.py", line 145, in check_connection
    sock.connect((self.server_host, self.server_port))
TimeoutError: timed out
[2025-01-23 01:20:27] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:27] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:29] students_app.windows.login - ERROR - Ошибка подключения к 192.168.0.164:9999: timed out
[2025-01-23 01:20:29] students_app.windows.login - ERROR - Подробная информация об ошибке:
Traceback (most recent call last):
  File "c:\Users\MrDes\OneDrive\Документы\GitHub\Pselu--Pself-\students_app\windows\login.py", line 145, in check_connection
    sock.connect((self.server_host, self.server_port))
TimeoutError: timed out
[2025-01-23 01:20:34] students_app.windows.login - DEBUG - Проверка подключения к 192.168.0.164:9999
[2025-01-23 01:20:34] students_app.windows.login - DEBUG - Сокет создан, устанавливаем соединение...
[2025-01-23 01:20:36] students_app.windows.login - ERROR - Ошибка подключения к 192.168.0.164:9999: timed out
[2025-01-23 01:20:36] students_app.windows.login - ERROR - Подробная информация об ошибке:
Traceback (most recent call last):
  File "c:\Users\MrDes\OneDrive\Документы\GitHub\Pselu--Pself-\students_app\windows\login.py", line 145, in check_connection
    sock.connect((self.server_host, self.server_port))
TimeoutError: timed out
